lib/config.py

```diff
@@ -15,11 +15,12 @@
     'com.widevine.alpha': 'widevine'
 }
 
-CDM_EXTENSIONS = (
-    '.so',
-    '.dll',
-    '.dylib'
-)
+WIDEVINE_CDM_FILENAME = {
+    'Android': 'libwidevinecdm.so',
```

Comment:
Shouldn't Android be `None`? As we don't manage it there.
The API should just skip any possible removal.

Reply:
Indeed.